### ISSUE TITLE
Masquer pages gestion et synchro manuelle en mode lecture anonyme

### ISSUE THREAD

**Issue by Simounet** [LeedRSS/Leed#469]
Je pense que nous devrions interdire l'accès à ces deux pages. Aucune action ne leur étant disponible, elles n'ont pas grand intérêt pour les utilisateurs anonymes. Elles laissent cependant transpirer pas mal d'informations.
L'idée serait de :
- n'afficher les liens 'Gestion' et 'Synchroniser' dans le header que si l'utilisateur est connecté
- modifier le comportement de `settings.php` pour qu'il ne soit plus accessible même via l'URL lorsque l'utilisateur n'est pas connecté
- nettoyer le code de `settings.php` pour enlever les exceptions vérifiant si l'utilisateur est connecté (et penser au traductions)

**Comment by Sbgodin** [comment]
Je suis d'accord. Toutefois, je ne vois pas où les traductions peuvent poser un soucis. Un accès à _settings.php_  échouera sur un genre de  __if (empty($myUser)) exit();__ et les traductions n'auront pas à être traitées.

**Comment by Simounet** [comment]
Dans le cas d'une utilisation en mode anonyme tu peux bien accéder à la page `settings.php` et chaque sous-partie renvoie `Vous devez être connecté pour effectuer cette action.`, `Vous devez être connecté pour voir les plugins.`, etc.
